internal/scti/handlers.go

```diff
@@ -68,48 +68,48 @@ var (
 	lastSCTTimestamp *prometheus.GaugeVec     // origin => value
 	reqsCounter      *prometheus.CounterVec   // origin, op => value
 	rspsCounter      *prometheus.CounterVec   // origin, op, code => value
-	rspLatency       *prometheus.HistogramVec // origin, op, code => value
+	rspDuration      *prometheus.HistogramVec // origin, op, code => value
```

Comment:
Out of curiosity, what's the thinking behind the name change? 
What I think this metric represents is called `latency distribution` in, er, other automatic monitoring systems we're intimately familiar with :)

If you're intent on changing it, "response duration" sounds a bit odd to my ears, but how about `rspTime`? Also, the description of the metric still explains it in terms of `Latency of responses`, so I guess that needs updating to match.

Reply:
Apologies, this PR lacked something very important: a link to https://prometheus.io/docs/practices/naming/, I've now added it. I'm 100% onboard with your comments and this looked a bit odd to me at first sight, being used to other monitoring systems... but it seems to be the prometheus way.

Reply:
I think the intent of the naming doc you linked was the `http_` prefix there, and the `..._request_duration_seconds` was just an example.

I found this "counter" (_hah!_) example in the [histogram section](https://prometheus.io/docs/tutorials/understanding_metric_types/#histogram) of the prom docs: 
> A common example would be the time it takes to reply to a request, called latency.

So I think there's latitude to fit in with styles we're familiar with.